src/common/errors.js

```diff
@@ -1,7 +1,18 @@
 'use strict';
 const util = require('util');
 
-class RippleError extends Error {
+// this is needed because extending builtins doesn't work in babel 6.x
+function extendableBuiltin(cls) {
+  function ExtendableBuiltin() {
+    cls.apply(this, arguments);
+  }
+  ExtendableBuiltin.prototype = Object.create(cls.prototype);
+  Object.setPrototypeOf(ExtendableBuiltin, cls);
+
+  return ExtendableBuiltin;
+}
+
+class RippleError extends extendableBuiltin(Error) {
```

Comment:
I think it would be cleaner to create a BaseError and extend that like this:

```javascript
function BaseError(message) {
  Error.captureStackTrace(this, this.constructor);

  Object.defineProperty(this, 'name', {
    get: () => this.constructor.name
  });
}
require('util').inherits(BaseError, Error);

class RippleError extends BaseError { /* ... */
```